sdk/eventhub/Azure.Messaging.EventHubs/src/Azure.Messaging.EventHubs.csproj

```diff
@@ -29,10 +29,6 @@
     </EmbeddedResource>
   </ItemGroup>
 
-  <ItemGroup>
-    <None Remove="Azure.Messaging.EventHubs.Reference.props" />
-  </ItemGroup>
```

Comment:
This is not longer a thing; it was added (and removed) as part of the checkpoint store package work.